.golangci.yaml

```diff
@@ -74,17 +75,15 @@ linters-settings:
     # it's a comma-separated list of prefixes
     local-prefixes: github.com/org/project
   govet:
-    # report about shadowed variables
-    check-shadowing: true
+    enable:
+      # report about shadowed variables
+      - shadow
   lll:
     # max line length, lines longer will be reported. Default is 120.
     # '\t' is counted as 1 character by default, and can be changed with the tab-width option
     line-length: 120
     # tab width in spaces. Default to 1.
     tab-width: 2
-  maligned:
-    # print struct with more effective memory layout or not, false by default
-    suggest-new: true
```

Comment:
This has not been working for quite some time, but could theoretically be replaced with the `fieldalignment` option from `govet`. However, I do wonder if the minimal number of bytes this saves is really that important compared to having ordered fields in a struct.